core/src/main/java/org/elasticsearch/action/get/GetResponse.java

```diff
@@ -168,4 +169,17 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         getResult.writeTo(out);
     }
+
+	@Override
+	public String toString() {
+		try {
```

Comment:
I think this is the same as `Strings.toString(this)`?

Otherwise I think this looks fine.